### YamlMime:ManagedReference
items:
  - uid: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception
    id: _multi_shard_aggregate_exception
    parent: com.microsoft.azure.elasticdb.query.exception
    children:
      - com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.getInnerExceptions()
      - com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException()
      - com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException(Exception)
      - com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException(List<Exception>)
      - com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException(String)
      - 'com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException(String,Exception)'
      - 'com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException(String,List<Exception>)'
      - com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.toString()
    href: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.yml
    langs:
      - java
    name: MultiShardAggregateException
    nameWithType: MultiShardAggregateException
    fullName: com.microsoft.azure.elasticdb.query.exception.MultiShardAggregateException
    type: Class
    source:
      remote: &ref_0
        path: elastic-db-tools/src/main/java/com/microsoft/azure/elasticdb/query/exception/MultiShardAggregateException.java
        branch: master
        repo: 'https://github.com/Microsoft/elastic-db-tools-for-java'
      path: elastic-db-tools/src/main/java/com/microsoft/azure/elasticdb/query/exception/MultiShardAggregateException.java
      startLine: 21
    package: com.microsoft.azure.elasticdb.query.exception
    summary: '<p><span data-ttu-id="e8fd7-101">シャード セット間でクエリを実行するときに発生した 1 つまたは複数の例外エラーを表します。</span><span class="sxs-lookup"><span data-stu-id="e8fd7-101">Represents one or more Exception errors that occurred when executing a query across a shard set.</span></span> <span data-ttu-id="e8fd7-102">InnerExceptions フィールドがこれらの例外を収集し、さらに検査または処理のため InnerExceptions を反復処理できる 1 つ。</span><span class="sxs-lookup"><span data-stu-id="e8fd7-102">The InnerExceptions field collects these exceptions and one can iterate through the InnerExceptions for further inspection or processing.</span></span> <span data-ttu-id="e8fd7-103">複数のシャード間で発生したエラーの情報をやり取りする目的: パブリック型</span><span class="sxs-lookup"><span data-stu-id="e8fd7-103">Purpose: Public type that communicates errors that occurred across multiple shards</span></span> </p>'
    syntax: &ref_1
      content: public class MultiShardAggregateException
    inheritance:
      - java.lang.Object
      - Exception
  - uid: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.getInnerExceptions()
    id: getInnerExceptions()
    parent: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception
    href: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.yml
    langs:
      - java
    name: getInnerExceptions()
    nameWithType: MultiShardAggregateException.getInnerExceptions()
    fullName: final List<Exception> com.microsoft.azure.elasticdb.query.exception.MultiShardAggregateException.getInnerExceptions()
    overload: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.getInnerExceptions*
    type: Method
    source:
      remote: *ref_0
      path: elastic-db-tools/src/main/java/com/microsoft/azure/elasticdb/query/exception/MultiShardAggregateException.java
      startLine: 108
    package: com.microsoft.azure.elasticdb.query.exception
    summary: <p><span data-ttu-id="e8fd7-104">現在の例外の原因となった例外インスタンスの読み取り専用コレクションを取得します。</span><span class="sxs-lookup"><span data-stu-id="e8fd7-104">Gets a read-only collection of the Exception instances that caused the current exception.</span></span> </p>
    syntax:
      content: public final List<Exception> getInnerExceptions()
      return:
        type: dd4ca536
  - uid: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException()
    id: MultiShardAggregateException()
    parent: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception
    href: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.yml
    langs:
      - java
    name: MultiShardAggregateException()
    nameWithType: MultiShardAggregateException.MultiShardAggregateException()
    fullName: com.microsoft.azure.elasticdb.query.exception.MultiShardAggregateException.MultiShardAggregateException()
    overload: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException*
    type: Constructor
    source:
      remote: *ref_0
      path: elastic-db-tools/src/main/java/com/microsoft/azure/elasticdb/query/exception/MultiShardAggregateException.java
      startLine: 28
    package: com.microsoft.azure.elasticdb.query.exception
    summary: <p><span data-ttu-id="e8fd7-105">新しいインスタンスを初期化、 <xref uid="com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception" data-throw-if-not-resolved="false">MultiShardAggregateException</xref>クラスです。</span><span class="sxs-lookup"><span data-stu-id="e8fd7-105">Initializes a new instance of the <xref uid="com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception" data-throw-if-not-resolved="false">MultiShardAggregateException</xref> class.</span></span> </p>
    syntax:
      content: public MultiShardAggregateException()
  - uid: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException(Exception)
    id: MultiShardAggregateException(Exception)
    parent: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception
    href: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.yml
    langs:
      - java
    name: MultiShardAggregateException(Exception innerException)
    nameWithType: MultiShardAggregateException.MultiShardAggregateException(Exception innerException)
    fullName: com.microsoft.azure.elasticdb.query.exception.MultiShardAggregateException.MultiShardAggregateException(Exception innerException)
    overload: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException*
    type: Constructor
    source:
      remote: *ref_0
      path: elastic-db-tools/src/main/java/com/microsoft/azure/elasticdb/query/exception/MultiShardAggregateException.java
      startLine: 50
    package: com.microsoft.azure.elasticdb.query.exception
    summary: |-
      <p><span data-ttu-id="e8fd7-106">新しいインスタンスを初期化、 <xref uid="com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception" data-throw-if-not-resolved="false">MultiShardAggregateException</xref>クラスです。</span><span class="sxs-lookup"><span data-stu-id="e8fd7-106">Initializes a new instance of the <xref uid="com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception" data-throw-if-not-resolved="false">MultiShardAggregateException</xref> class.</span></span></p>
      <p></p>
    syntax:
      content: public MultiShardAggregateException(Exception innerException)
      parameters:
        - id: innerException
          type: 8a99d4b0
          description: <p><span data-ttu-id="e8fd7-107">現在の例外の原因となった例外</span><span class="sxs-lookup"><span data-stu-id="e8fd7-107">The Exception that caused the current exception</span></span> </p>
  - uid: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException(List<Exception>)
    id: MultiShardAggregateException(List<Exception>)
    parent: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception
    href: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.yml
    langs:
      - java
    name: MultiShardAggregateException(List<Exception> innerExceptions)
    nameWithType: MultiShardAggregateException.MultiShardAggregateException(List<Exception> innerExceptions)
    fullName: com.microsoft.azure.elasticdb.query.exception.MultiShardAggregateException.MultiShardAggregateException(List<Exception> innerExceptions)
    overload: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException*
    type: Constructor
    source:
      remote: *ref_0
      path: elastic-db-tools/src/main/java/com/microsoft/azure/elasticdb/query/exception/MultiShardAggregateException.java
      startLine: 77
    package: com.microsoft.azure.elasticdb.query.exception
    summary: |-
      <p><span data-ttu-id="e8fd7-108">新しいインスタンスを初期化、 <xref uid="com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception" data-throw-if-not-resolved="false">MultiShardAggregateException</xref>クラスです。</span><span class="sxs-lookup"><span data-stu-id="e8fd7-108">Initializes a new instance of the <xref uid="com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception" data-throw-if-not-resolved="false">MultiShardAggregateException</xref> class.</span></span></p>
      <p></p>
    syntax:
      content: public MultiShardAggregateException(List<Exception> innerExceptions)
      parameters:
        - id: innerExceptions
          type: 5c5d6e37
          description: <p><span data-ttu-id="e8fd7-109">現在の例外の原因となった例外の一覧</span><span class="sxs-lookup"><span data-stu-id="e8fd7-109">A list of Exception that caused the current exception</span></span> </p>
  - uid: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException(String)
    id: MultiShardAggregateException(String)
    parent: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception
    href: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.yml
    langs:
      - java
    name: MultiShardAggregateException(String message)
    nameWithType: MultiShardAggregateException.MultiShardAggregateException(String message)
    fullName: com.microsoft.azure.elasticdb.query.exception.MultiShardAggregateException.MultiShardAggregateException(String message)
    overload: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException*
    type: Constructor
    source:
      remote: *ref_0
      path: elastic-db-tools/src/main/java/com/microsoft/azure/elasticdb/query/exception/MultiShardAggregateException.java
      startLine: 38
    package: com.microsoft.azure.elasticdb.query.exception
    summary: |-
      <p><span data-ttu-id="e8fd7-110">新しいインスタンスを初期化、 <xref uid="com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception" data-throw-if-not-resolved="false">MultiShardAggregateException</xref>クラスです。</span><span class="sxs-lookup"><span data-stu-id="e8fd7-110">Initializes a new instance of the <xref uid="com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception" data-throw-if-not-resolved="false">MultiShardAggregateException</xref> class.</span></span></p>
      <p></p>
    syntax:
      content: public MultiShardAggregateException(String message)
      parameters:
        - id: message
          type: '26831127'
          description: <p><span data-ttu-id="e8fd7-111">例外の原因を説明するエラー メッセージ</span><span class="sxs-lookup"><span data-stu-id="e8fd7-111">The error message that explains the reason for the exception</span></span> </p>
  - uid: 'com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException(String,Exception)'
    id: 'MultiShardAggregateException(String,Exception)'
    parent: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception
    href: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.yml
    langs:
      - java
    name: 'MultiShardAggregateException(String message, Exception innerException)'
    nameWithType: 'MultiShardAggregateException.MultiShardAggregateException(String message, Exception innerException)'
    fullName: 'com.microsoft.azure.elasticdb.query.exception.MultiShardAggregateException.MultiShardAggregateException(String message, Exception innerException)'
    overload: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException*
    type: Constructor
    source:
      remote: *ref_0
      path: elastic-db-tools/src/main/java/com/microsoft/azure/elasticdb/query/exception/MultiShardAggregateException.java
      startLine: 62
    package: com.microsoft.azure.elasticdb.query.exception
    summary: |-
      <p><span data-ttu-id="e8fd7-112">新しいインスタンスを初期化、 <xref uid="com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception" data-throw-if-not-resolved="false">MultiShardAggregateException</xref>クラスです。</span><span class="sxs-lookup"><span data-stu-id="e8fd7-112">Initializes a new instance of the <xref uid="com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception" data-throw-if-not-resolved="false">MultiShardAggregateException</xref> class.</span></span></p>
      <p></p>
    syntax:
      content: 'public MultiShardAggregateException(String message, Exception innerException)'
      parameters:
        - id: message
          type: '26831127'
          description: <p><span data-ttu-id="e8fd7-113">例外の原因を説明するエラー メッセージ</span><span class="sxs-lookup"><span data-stu-id="e8fd7-113">The error message that explains the reason for the exception</span></span> </p>
        - id: innerException
          type: 8a99d4b0
          description: <p><span data-ttu-id="e8fd7-114">現在の例外の原因となった例外の一覧</span><span class="sxs-lookup"><span data-stu-id="e8fd7-114">A list of Exception that caused the current exception</span></span> </p>
  - uid: 'com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException(String,List<Exception>)'
    id: 'MultiShardAggregateException(String,List<Exception>)'
    parent: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception
    href: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.yml
    langs:
      - java
    name: 'MultiShardAggregateException(String message, List<Exception> innerExceptions)'
    nameWithType: 'MultiShardAggregateException.MultiShardAggregateException(String message, List<Exception> innerExceptions)'
    fullName: 'com.microsoft.azure.elasticdb.query.exception.MultiShardAggregateException.MultiShardAggregateException(String message, List<Exception> innerExceptions)'
    overload: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException*
    type: Constructor
    source:
      remote: *ref_0
      path: elastic-db-tools/src/main/java/com/microsoft/azure/elasticdb/query/exception/MultiShardAggregateException.java
      startLine: 91
    package: com.microsoft.azure.elasticdb.query.exception
    summary: |-
      <p><span data-ttu-id="e8fd7-115">新しいインスタンスを初期化、 <xref uid="com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception" data-throw-if-not-resolved="false">MultiShardAggregateException</xref>クラスです。</span><span class="sxs-lookup"><span data-stu-id="e8fd7-115">Initializes a new instance of the <xref uid="com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception" data-throw-if-not-resolved="false">MultiShardAggregateException</xref> class.</span></span></p>
      <p></p>
    syntax:
      content: 'public MultiShardAggregateException(String message, List<Exception> innerExceptions)'
      parameters:
        - id: message
          type: '26831127'
          description: <p><span data-ttu-id="e8fd7-116">例外の原因を説明するエラー メッセージ</span><span class="sxs-lookup"><span data-stu-id="e8fd7-116">The error message that explains the reason for the exception</span></span> </p>
        - id: innerExceptions
          type: 5c5d6e37
          description: <p><span data-ttu-id="e8fd7-117">現在の例外の原因となった例外の一覧</span><span class="sxs-lookup"><span data-stu-id="e8fd7-117">A list of Exception that caused the current exception</span></span> </p>
    exceptions:
      - type: 3c87bd19
        description: <p><span data-ttu-id="e8fd7-118"><em>InnerExceptions</em>が null です</span><span class="sxs-lookup"><span data-stu-id="e8fd7-118">The <em>innerExceptions</em>  is null</span></span> </p>
  - uid: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.toString()
    id: toString()
    parent: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception
    href: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.yml
    langs:
      - java
    name: toString()
    nameWithType: MultiShardAggregateException.toString()
    fullName: String com.microsoft.azure.elasticdb.query.exception.MultiShardAggregateException.toString()
    overload: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.toString*
    type: Method
    source:
      remote: *ref_0
      path: elastic-db-tools/src/main/java/com/microsoft/azure/elasticdb/query/exception/MultiShardAggregateException.java
      startLine: 117
    package: com.microsoft.azure.elasticdb.query.exception
    summary: <p><span data-ttu-id="e8fd7-119">その内部の例外を含むこの例外の文字列表現を提供します。</span><span class="sxs-lookup"><span data-stu-id="e8fd7-119">Provides a string representation of this exception including its inner exceptions.</span></span> </p>
    syntax:
      content: public String toString()
      return:
        type: '26831127'
references:
  - uid: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.MultiShardAggregateException*
    name: MultiShardAggregateException
    nameWithType: MultiShardAggregateException.MultiShardAggregateException
    fullName: com.microsoft.azure.elasticdb.query.exception.MultiShardAggregateException.MultiShardAggregateException
    package: com.microsoft.azure.elasticdb.query.exception
  - uid: '26831127'
    spec.java:
      - name: String
        fullName: String
  - uid: 8a99d4b0
    spec.java:
      - name: Exception
        fullName: Exception
  - uid: 5c5d6e37
    spec.java:
      - name: List<Exception>
        fullName: List<Exception>
  - uid: 3c87bd19
    spec.java:
      - name: IllegalArgumentException
        fullName: IllegalArgumentException
  - uid: dd4ca536
    spec.java:
      - name: final List<Exception>
        fullName: final List<Exception>
  - uid: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.getInnerExceptions*
    name: getInnerExceptions
    nameWithType: MultiShardAggregateException.getInnerExceptions
    fullName: final List<Exception> com.microsoft.azure.elasticdb.query.exception.MultiShardAggregateException.getInnerExceptions
    package: com.microsoft.azure.elasticdb.query.exception
  - uid: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.toString*
    name: toString
    nameWithType: MultiShardAggregateException.toString
    fullName: String com.microsoft.azure.elasticdb.query.exception.MultiShardAggregateException.toString
    package: com.microsoft.azure.elasticdb.query.exception
  - uid: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception
    parent: com.microsoft.azure.elasticdb.query.exception
    href: com.microsoft.azure.elasticdb.query.exception._multi_shard_aggregate_exception.yml
    name: MultiShardAggregateException
    nameWithType: MultiShardAggregateException
    fullName: com.microsoft.azure.elasticdb.query.exception.MultiShardAggregateException
    type: Class
    summary: '<p><span data-ttu-id="e8fd7-120">シャード セット間でクエリを実行するときに発生した 1 つまたは複数の例外エラーを表します。</span><span class="sxs-lookup"><span data-stu-id="e8fd7-120">Represents one or more Exception errors that occurred when executing a query across a shard set.</span></span> <span data-ttu-id="e8fd7-121">InnerExceptions フィールドがこれらの例外を収集し、さらに検査または処理のため InnerExceptions を反復処理できる 1 つ。</span><span class="sxs-lookup"><span data-stu-id="e8fd7-121">The InnerExceptions field collects these exceptions and one can iterate through the InnerExceptions for further inspection or processing.</span></span> <span data-ttu-id="e8fd7-122">複数のシャード間で発生したエラーの情報をやり取りする目的: パブリック型</span><span class="sxs-lookup"><span data-stu-id="e8fd7-122">Purpose: Public type that communicates errors that occurred across multiple shards</span></span> </p>'
    syntax: *ref_1